src/index.ts

```diff
@@ -63,6 +63,8 @@ export function config(callbacks: Partial<WalletActionsCallbacks>) {
 }
 
 async function authWithProvider(loginProvider: Web3AuthLoginProvider) {
+  // instead of exposing fn for unauth, we just logout user every time before he tries to log in
```

Comment:
`instead of exposing fn for unauth, we just logout the user just after they logged in` would be more accurate